eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/ZeroConfAliasIntegrationSpec.scala

```diff
@@ -98,44 +98,38 @@ class ZeroConfAliasIntegrationSpec extends FixtureSpec with IntegrationPatience
       }
     }
 
-    eventually {
```

Comment:
I believe we can now revert the changes to that file, right?

Reply:
I did, but we can't remove pre-existing `eventually`.